src/resolver.js

```diff
@@ -20,19 +20,23 @@ export function clearCache() {
   plugins.refs.clearCache()
 }
 
-export default function resolve({http, fetch, spec, url, mode, allowMetaPatches = true}) {
+export default function resolve({http, fetch, spec, url, baseDoc, mode, allowMetaPatches = true}) {
+  // @TODO Swagger-UI uses baseDoc instead of url, this is to allow both
+  // need to fix and pick one.
+  baseDoc = baseDoc || url
```

Comment:
I would do the reverse.
`url = url || baseDoc` with a note saying `baseDoc` is deprecated.